scripts/make/update.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+script_path=$(dirname $0)
+working_dir=$(pwd)
+cd "$script_path"
+cd ../..
+repo_root=$(pwd)
+
+source "$repo_root/.build.env"
+
+if [ "$drupal_version" == "7" ]; then
+  drush @docker rr --no-cache-clear \
+    && drush @docker updb -y \
+    && drush @docker en -y master \
+    && drush @docker cc drush \
+    && drush @docker master-exec -y \
+    && drush @docker cc all \
+    && drush @docker cc drush \
+    && drush @docker fl
```

Comment:
What does `drush fl` do?

Reply:
It prints a list of the sites Drupal Features and their current state.
Its useful after an update so you can see if any features are overridden - which they shouldn't be ever according to best practice.
In reality sometimes its easier said that done to avoid some features being overridden, but at least being visually made aware that features are overridden keeps it front-and-centre in our mind that when we have an opportunity we should make efforts to fix features that are in an overridden state.